### PR TITLE
Add AMP Optimizer Guide

### PR DESCRIPTION
**Pull request by sebastianbenz**
This adds a new landing page for AMP Optimizers. This guide is split into multiple parts and is going to live under the Optimize & Measure sectio:

* Using an AMP Optimizer (intro and overview)
** How does an AMP Optimizer work
** Using the Node.js AMP Optimizer (tutorial)

This guide will replace the existing server-side rendering guide on amp.dev

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/node-amp-optimizer.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+$title: Node.js AMP Optimizer Guide
+$order: 1
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/node-amp-optimizer.md:3]: Both this one and `How an AMP Optimizer works` are ordered at 1 - is that on purpose?

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/node-amp-optimizer.md:3]: that's just me forgetting how stuff works...fixed

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+author: sebastianbenz
+---
+
+AMP Optimizers are tools that bring AMP Cache optimizations to your own site. Using an AMP Optimizer is key to creating a great [page experience](https://developers.google.com/search/docs/guides/page-experience) and achieving [Core Web Vitals](https://web.dev/vitals/) compliance. If you want to learn more about how an AMP Optimizer works, checkout our [detailed AMP Optimizations Guide](explainer.md).
+
+## Isn’t AMP already fast?
+
+In the beginning, AMP caches served the majority of AMP pages. These caches performed additional optimizations on pages to guarantee a strong user experience. But, over time, more surfaces started linking to AMP pages and developers began building entire websites with AMP.
+
+To ensure best performance and user experience for AMP pages served on the origin. the AMP team
```
Comment (patrickkettner) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/index.md:18]: two nits
1: I believe it should be "To ensure the best performance"
2. Is the second sentence a valediction or just cut off?

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/index.md:18]: No idea what I was thinking there. Definitely not intended...

**Pull request merged**